### ISSUE TITLE
Is ansible-node-exporter abandoned? No commits have been made since January 2022 (now December 2022)

### ISSUE THREAD

**Issue by louwrentius** [cloudalchemy/ansible-node-exporter#279]
Similar to my question in October 2022 about ansible-prometheus, I'm wondering if this repo is abandoned like ansible-prometheus.

**Comment by SuperQ** [comment]
Yes, we're moving things to https://github.com/prometheus-community/ansible

**Comment by louwrentius** [comment]
Thank you for sharing! 

If/When it suits you, maybe can this be added in the readme.md at the top for  others to find, maybe for all relevant cloudalchemy repos?

Anyway, thanks!

**Comment by jantari** [comment]
Another thing, the README at https://github.com/prometheus-community/ansible/blob/main/roles/node_exporter/README.md states that you've had to move everything to https://galaxy.ansible.com/cloudalchemy/node_exporter which is this repo once again ( https://github.com/cloudalchemy/ansible-node-exporter ).

So currently it seems there's a circular deprecation where each repository tells me ot use the other instead.

**Comment by antonr-p2p** [comment]
You can find fixed version in https://github.com/prometheus-community/ansible/tree/main/roles/node_exporter

**Comment by SuperQ** [comment]
**This role has been deprecated in favor of a the [prometheus-community/ansible](https://github.com/prometheus-community/ansible) collection.**
